### DISCUSSION TITLE
Flutter 的macos平台应该如何集成插件呢？

### DISCUSSION THREAD

**Discussion by Grieey** [Tencent/libpag#1568] [General] [answered]
目前的插件支持了ios和android，想在macos平台上也进行支持，应该如何集成？

**Comment by kevingpqi123** [comment, marked as answer]
Flutter 平台是公司内其他团队适配的，目前仅适配了 Android 和 iOS，由于我们自身没有该业务场景的需求，如果需要其它平台，可能需要业务方参考 iOS 平台自己适配，欢迎参与共建
